fw-child/resources/js/child-functions.js

```diff
@@ -708,6 +709,25 @@ const $ = jQuery;
 
     };
 
+    //
+    // Dynamic logo block display scroll behavior
+    //
+    //
+
+    if ($('#floating-header').length && $('#hero').length) {
+      $(window).scroll(function() {
+        const floatingHeaderBottom = $('#floating-header').offset().top + $('#floating-header').outerHeight();
+        const elementAfterHero = $('#hero').next();
+        const elementAfterHeroTop = elementAfterHero.offset().top;
+
+        if (floatingHeaderBottom >= elementAfterHeroTop && !elementAfterHero.hasClass('query-page')) {
+          $('#floating-header').addClass('on-white');
+        } else {
+          $('#floating-header').removeClass('on-white');
+        }
+      });
+    }
+
```

Comment:
I think i would trigger the color change in the middle of the icon or so to avoid this:
![Capture d’écran, le 2024-08-22 à 14 32 33](https://github.com/user-attachments/assets/98f8237f-6cb4-4295-b6a9-4f99ae3dccef)